apps/ewallet_db/lib/ewallet_db/transaction_consumption.ex

```diff
@@ -1,18 +1,19 @@
-defmodule EWalletDB.TransactionRequestConsumption do
+defmodule EWalletDB.TransactionConsumption do
```

Comment:
**Request:** The schema name still stays the same `"transaction_request_consumption"`?